src/commands/apps-sdkconfig.ts

```diff
@@ -0,0 +1,128 @@
+import * as ora from "ora";
+import * as fs from "fs-extra";
+
+import * as Command from "../command";
+import {
+  AppMetadata,
+  AppPlatform,
+  getAppConfig,
+  getAppPlatform,
+  listFirebaseApps,
+} from "../management/apps";
+import * as FirebaseError from "../error";
+import * as requireAuth from "../requireAuth";
+import * as logger from "../logger";
+import { promptOnce } from "../prompt";
+
+async function selectAppInteractively(
+  projectId: string,
+  appPlatform: AppPlatform
+): Promise<AppMetadata> {
+  if (!projectId) {
+    throw new FirebaseError("Project ID must not be empty.");
+  }
+
+  const apps: AppMetadata[] = await listFirebaseApps(projectId, appPlatform);
+  if (apps.length === 0) {
+    throw new FirebaseError(
+      `There are no ${appPlatform === AppPlatform.ANY ? "" : appPlatform + " "}apps ` +
+        "associated with this Firebase project"
+    );
+  }
+
+  const choices = apps.map((app: any) => {
+    return {
+      name:
+        `${app.displayName || app.bundleId || app.packageName}` +
+        ` - ${app.appId} (${app.platform})`,
+      value: app,
+    };
+  });
+
+  return await promptOnce({
+    type: "list",
+    name: "id",
+    message:
+      `Select the ${appPlatform === AppPlatform.ANY ? "" : appPlatform + " "}` +
+      "app to get the configuration data:",
+    choices,
+  });
+}
+
+module.exports = new Command("apps:sdkconfig [platform] [appId]")
+  .description(
+    "print the Google Services config of a Firebase app. " +
+      "[platform] can be IOS, ANDROID or WEB (case insensitive)"
+  )
+  .option("-o, --out [file]", "(optional) write config output to a file")
```

Comment:
If `file` is required, I believe is has to be in angle brackets: `<file>`

Reply:
`file` is optional argument for the option. Default file name will be used if only `--out` is provided (in which case `options.out === true`).
The default file names are:
**iOS** GoogleService-Info.plist
**Android** google-services.json
**Web**  google-config.js

Initially, I thought that we don't have support for `--option [optionalArgument]`, but I did test my command and this seems to work properly